src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -1067,7 +1067,7 @@ public function getSqlType(Literal|string $type, ?int $limit = null): array
             case static::PHINX_TYPE_DOUBLE:
                 return ['name' => 'double precision'];
             case static::PHINX_TYPE_STRING:
-                return ['name' => 'character varying', 'limit' => 255];
+                return ['name' => 'character varying'];
```

Comment:
Would it make more sense to generate a `text` data type for postgres for this?

Reply:
The text type works fine for PostgreSQL, but we are used to using it for multi-line inputs and character variing for single-line inputs (this is also the default for CakePHP for forms).

PostgreSQL directly recommends not to restrict this for no reason, it supposedly has no performance impact.
See: https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_varchar.28n.29_by_default